policy-controller/Cargo.toml

```
@@ -23,6 +25,7 @@ linkerd-policy-controller-core = { path = "./core" }
 linkerd-policy-controller-grpc = { path = "./grpc" }
 linkerd-policy-controller-k8s-index = { path = "./k8s/index" }
 linkerd-policy-controller-k8s-api = { path = "./k8s/api" }
+openssl = "0.10.45"
```

This should be optional so it's only enabled when the feature is set.

```suggestion
openssl = { version = "0.10.45", optional = true }
```